students/arkadiusz_kasprzyk/lesson_02_flow_control/the_number_of_elements_equal_maximum.py

```diff
@@ -0,0 +1,30 @@
+"""
+description:
+    A sequence consists of integer numbers and ends with the number 0.
+    Determine how many elements of this sequence are equal to its largest element.
+"""
+
+print("""
+    A sequence consists of integer numbers and ends with the number 0. 
+    Determines how many elements of this sequence are equal to its largest element.
+""")
+
+n = int(input("give the first number : "))
+
+max = n
```

Comment:
`max` is also a built-in. By convention you can either append `_` at the end like `max_` or use some qualifier for readability `max_number` or go for `maximum`